[2022-06-06 15:13:53,172] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-06T18:13:34.330178+00:00 [queued]>
[2022-06-06 15:13:53,191] {taskinstance.py:1160} INFO - Dependencies all met for <TaskInstance: census.populate_table scheduled__2022-06-06T18:13:34.330178+00:00 [queued]>
[2022-06-06 15:13:53,192] {taskinstance.py:1357} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 15:13:53,192] {taskinstance.py:1358} INFO - Starting attempt 1 of 1
[2022-06-06 15:13:53,192] {taskinstance.py:1359} INFO - 
--------------------------------------------------------------------------------
[2022-06-06 15:13:53,216] {taskinstance.py:1378} INFO - Executing <Task(PythonOperator): populate_table> on 2022-06-06 18:13:34.330178+00:00
[2022-06-06 15:13:53,226] {standard_task_runner.py:52} INFO - Started process 16870 to run task
[2022-06-06 15:13:53,231] {standard_task_runner.py:79} INFO - Running: ['airflow', 'tasks', 'run', 'census', 'populate_table', 'scheduled__2022-06-06T18:13:34.330178+00:00', '--job-id', '1217', '--raw', '--subdir', 'DAGS_FOLDER/dag.py', '--cfg-path', '/tmp/tmpsown41hs', '--error-file', '/tmp/tmpvj4fg7ac']
[2022-06-06 15:13:53,232] {standard_task_runner.py:80} INFO - Job 1217: Subtask populate_table
[2022-06-06 15:13:53,319] {task_command.py:370} INFO - Running <TaskInstance: census.populate_table scheduled__2022-06-06T18:13:34.330178+00:00 [running]> on host uiliam-Vostro-15-3515
[2022-06-06 15:13:53,434] {taskinstance.py:1570} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=census
AIRFLOW_CTX_TASK_ID=populate_table
AIRFLOW_CTX_EXECUTION_DATE=2022-06-06T18:13:34.330178+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-06-06T18:13:34.330178+00:00
[2022-06-06 15:13:54,248] {python.py:173} INFO - Done. Returned value was: 1630
[2022-06-06 15:13:54,278] {taskinstance.py:1396} INFO - Marking task as SUCCESS. dag_id=census, task_id=populate_table, execution_date=20220606T181334, start_date=20220606T181353, end_date=20220606T181354
[2022-06-06 15:13:54,327] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-06 15:13:54,394] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
